packages/storybook/cxl-ui/cxl-vaadin-accordion/theme=cxl-faq.stories.js

```diff
@@ -0,0 +1,62 @@
+import { html } from 'lit-html';
+import '@conversionxl/cxl-ui/src/components/cxl-vaadin-accordion.js';
```

Comment:
Story needs a data file.

* Create REST API app key in your staging and/or live WP profile
* Use IntelliJ REST API client to fetch data like

```
GET https://<server>/wp-json/wp/v2/posts
Accept: application/json
Authorization: Basic anna-maria <password>
```

Modify API URL to needs, of course.

Reply:
@lkraav I got the response working but I don't know the URL there I could get the correct data

Reply:
Which WP REST API training have you taken?

Reply:
no trainings. 
But I created the new pods in staging and got the url and data